src/Mod/Path/App/Area.cpp

```diff
@@ -864,14 +835,13 @@ struct WireJoiner {
     // This algorithm tries to find a set of closed wires that includes as many
     // edges (added by calling add() ) as possible. One edge may be included
     // in more than one closed wires if it connects to more than one edges.
-    int findClosedWires() {
+    int findClosedWires(double tol = Precision::Confusion()) {
```

Comment:
Make wire matching tolerance configurable, and effectively change the default to be equal to AreaParams.Tolerance, which defaults to Precision::Confusion().